### DISCUSSION TITLE
snowbridge_Error

### DISCUSSION THREAD

**Discussion by firebeegit** [Snowfork/snowbridge#967] [Q&A]
Hi Team,
we are trying to snowbridge running in my local system . I got error during this command (cd core && pnpm install) in this link I followed ( https://docs.snowbridge.network/other/quick-start-guide ) . let me know any solution . I attached screenshot to this message.

Thank you,

![image](https://github.com/Snowfork/snowbridge/assets/141648418/d5a2dd43-cd05-4943-b99a-d1a0a761174f)

**Comment by claravanstaden** [comment]
Hey @firebeegit! That guide is fairly outdated. You can use [these steps instead](https://github.com/Snowfork/snowbridge#development). They are fairly straightforward and not many steps are required.

**Comment by firebeegit** [comment]
Hi @claravanstaden 
Thanks for your response,
I'm followed your reference steps in my local system . I got error during this command ( nix build ) in this link I followed (https://github.com/Snowfork/snowbridge#development) . let me know any solution . I attached error screenshot to this message.

Thank you,
![error1](https://github.com/Snowfork/snowbridge/assets/141648418/d3bd4829-2033-47fd-b301-ea2523efd6a7)

**Reply by claravanstaden** [reply]
@firebeegit it would be best to Google and resolve that error yourself - it seems related to running Nix on your local setup and because of not Snowbridge itself.

**Reply by firebeegit** [reply]
Hi @claravanstaden ,
Thanks for your response,
In my system Linux(Ubuntu-22.04). Please let me know if possible to work your reference code in Linux(Ubuntu-22.04) system?

**Reply by claravanstaden** [reply]
Yes, definitely. I see you ran `nix build`. It should be `nix develop`.
